setup.py

```diff
@@ -6,6 +6,9 @@
       author="Ewen Cheslack-Postava",
       author_email='ewen@confluent.io',
       platforms=["any"], 
+      entry_points={
+        'console_scripts': ['ducktape=ducktape.__main__:main'],
```

Comment:
I'm fairly sure it can be an arbitrary module.

I guess the advantage is that by using __main__.py, I can run ducktape with either command
python -m ducktape
ducktape